tests/test_aurora.py

```diff
@@ -0,0 +1,434 @@
+import pytest
+import torch
+from graph_weather.models.aurora.model import AuroraModel, EarthSystemLoss
+from graph_weather.models.aurora.encoder import Swin3DEncoder
+from graph_weather.models.aurora.decoder import Decoder3D
+from graph_weather.models.aurora.processor import PerceiverProcessor
+
+
+@pytest.fixture
+def sample_3d_data():
+    """Create small sample 3D data for testing"""
+    batch_size = 2
+    channels = 1
+    depth = height = width = 8  # Reduced from 32 to 8
+    return torch.randn(batch_size, channels, depth, height, width)
+
+
+@pytest.fixture
+def swin3d_config():
+    """Minimal configuration for Swin3D encoder"""
+    return {"in_channels": 1, "embed_dim": 32}  # Reduced from 96 to 32
+
+
+@pytest.fixture
+def sample_unstructured_data():
+    """Create sample unstructured point data for testing"""
+    batch_size = 1
+    num_points = 100
+    num_features = 4
+
+    points = torch.rand(batch_size, num_points, 2) * 360
+    points[:, :, 0] = points[:, :, 0] - 180
+    points[:, :, 1] = points[:, :, 1] - 90
+
+    features = torch.randn(batch_size, num_points, num_features - 2)
+
+    return points, features
+
+
+@pytest.fixture
+def model_config():
+    """Basic model configuration for unstructured point data"""
+    return {
+        "input_features": 2,
+        "output_features": 2,
+        "embed_dim": 24,
+        "latent_dim": 128,
+        "max_points": 5000,
+        "max_seq_len": 1024,
+    }
```

Comment:
Really nice tests! One thing I would suggest, to ensure the tests are as quick to run as possible, would be to make this smaller
```suggestion
    """Basic model configuration for unstructured point data"""
    return {
        "input_features": 2,
        "output_features": 2,
        "embed_dim": 6,
        "latent_dim": 16
        "max_points": 50,
        "max_seq_len": 128,
    }
```

Or something like that.